CHANGELOG.md

```diff
@@ -25,6 +25,10 @@ Changes are grouped as follows
 - Support for the `/simulators` and `/simulators/integration` API endpoints.
 
 
+## [7.73.10] - 2025-04-01
+### Fixed
+- Iterating through datapoints would max out at 100k values total for any instance IDs used.
```

Comment:
Don't understand this. It is no longer maxing out after 100k values?

Reply:
Correct. Open for suggestions on wording

Reply:
Rewrote it now, let me know if makes more sense